code/datums/uplink_item.dm

```diff
@@ -1570,14 +1570,14 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	name = "Thermal Vision Implant"
 	desc = "These cybernetic eyes will give you thermal vision. Comes with an automated implanting tool."
 	reference = "CIT"
-	item = /obj/item/organ/internal/cyberimp/eyes/thermals
+	item = /obj/item/organ/internal/eyes/cybernetic/thermals
```

Comment:
Auto-implanters, which come with the implant kits nukies get, only accept subtypes of /obj/item/organ/internal/cyberimp  and not anything else, so either nukies will need to do surgery to install their new eyes, or autoimplanter will need to be changed to accept normal organs